src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1885,7 +1884,7 @@ export class StudyViewPageStore {
                         !attributeChanged) {
                         result = this.initialVisibleAttributesClinicalDataBinCountData.result;
                     } else {
-                        if (this._clinicalDataIntervalFilterSet.has(uniqueKey)) {
+                        if (this._clinicalDataIntervalFilterSet.has(uniqueKey) || this.isInitialFilterState) {
```

Comment:
The await might need to be updated as well.

Reply:
Looks like its already present